crates/wash-lib/src/build/component.rs

```diff
@@ -97,7 +35,7 @@ pub fn build_actor(
     actor_config: &ActorConfig,
     language_config: &LanguageConfig,
     common_config: &CommonConfig,
-    signing_config: Option<SignConfig>,
+    signing_config: Option<&SignConfig>,
```

Comment:
What was the reason behind the changes for borrowing `SignConfig` in the function signatures now? Doesn't really matter since this is the CLI, but I was curious

Reply:
Primarily for consistency, this is in the library so I figured that depending on the usage it would be useful to be able to pass a reference instead of an owned config, just like the other functions